server/app/models/fusor/deployment.rb

```diff
@@ -34,7 +34,7 @@ class Deployment < ActiveRecord::Base
     has_many :deployment_hosts, :class_name => "Fusor::DeploymentHost", :foreign_key => :deployment_id, :inverse_of => :deployment
     has_many :discovered_hosts, :through => :deployment_hosts, :foreign_key => :discovered_host_id, :source => :discovered_host
     alias_attribute :discovered_host_id, :rhev_engine_host_id
-    alias_attribute :foreman_task_id, :foreman_task_uuid
+    attr_accessor :foreman_task_id
```

Comment:
having `alias_attribute` was the cause of bug, since `foreman_task_id` was passed as `nil` so it overrode `foreman_task_uuid` as `nil`
having `attr_accessor` prevents the error from ember sending `foreman_task_id`